surrealdb/clients/synchronous.py

```diff
@@ -0,0 +1,169 @@
+"""
+Copyright © SurrealDB Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from json import JSONDecodeError
+from types import TracebackType
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Type
+
+import httpx
+
+from ..common import json as jsonlib
+from ..exceptions.surreal_exception import SurrealException
+from ..models.response import SurrealResponse
+
+__all__ = ("SurrealDBClient",)
+
+
+class SurrealDBClient:
```

Comment:
naming this `SynchronousSurrealDBClient` would probably better, especially since it could be removed in the future.

Reply:
renamed to `SyncSurrealDBClient`